lerna.json

```diff
@@ -1,6 +1,8 @@
 {
-    "packages": ["packages/*"],
-    "version": "1.0.0",
+    "packages": [
+        "packages/*"
+    ],
+    "version": "1.1.6",
```

Comment:
This Version

Reply:
updated to 0.1.1 to match the npm deployment